### ISSUE TITLE
[jGL] support glHint

### ISSUE THREAD

**Issue by laeubi** [jzy3d/jzy3d-api#194]
While adapting an example to jGL i noticed that the function `glHint` is missing like in the following example:

` gl.glHint(GL.GL_PERSPECTIVE_CORRECTION_HINT, gl.GL_NICEST);`

as these are provider specific hints (see https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glHint.xhtml) the best might be to make this an (int, int) so extensions can choose to support more properties.

For AWT for example it might be used to set the rendering hints.

**Comment by jzy3d** [comment]
Indeed, glHint is defined as of OpenGL 2 - while jGL only implements OpenGL 1.

**Comment by laeubi** [comment]
It still might be useful to implement it anyways as its not much effort and will allow some kind of cross-window-kit way to give some hints about render quality.
